documents/payment.md

```diff
@@ -0,0 +1,467 @@
+  "email": "testme@acme.com",
+  "notes": {
+    "key1": "value1",
+    "key2": "value2"
+  },
+  "fee": null,
+  "tax": null,
+  "error_code": null,
+  "error_description": null,
+  "error_source": null,
+  "error_step": null,
+  "error_reason": null,
+  "acquirer_data": {
+    "bank_transaction_id": "0125836177"
+  },
+  "created_at": 1553504328
+}
+```
+-------------------------------------------------------------------------------------------------------
+
+### Fetch expanded card or emi details for payments
+
+Request #1: Card
+
+```rb
+Razorpay::Payment.all({'expand[]':'card'})
+```
+
+Request #2: EMI
+
+```rb
+Razorpay::Payment.all({'expand[]':'emi'})
+```
+
+**Response:**<br>
+For expanded card or emi details for payments response please click [here](https://razorpay.com/docs/api/payments/#fetch-expanded-card-or-emi-details-for-payments)
+
+-------------------------------------------------------------------------------------------------------
+
+### Fetch card details with paymentId
+
+```rb
+Razorpay::Payment.fetch_card_details(paymentId)
+```
+
+**Parameters:**
+
+| Name        | Type    | Description                          |
+|-------------|---------|--------------------------------------|
+| paymentId* | string  | Id of the payment to update          |
+
+**Response:**
+```json
+{
+  "id": "card_6krZ6bcjoeqyV9",
+  "entity": "card",
+  "name": "Gaurav",
+  "last4": "3335",
+  "network": "Visa",
+  "type": "debit",
+  "issuer": "SBIN",
+  "international": false,
+  "emi": null,
+  "sub_type": "business"
+}
+```
+-------------------------------------------------------------------------------------------------------
+
+### Fetch Payment Downtime Details
+
+```rb
+Razorpay::Payment.fetch_payment_downtime
+```
+**Response:** <br>
+For payment downtime response please click [here](https://razorpay.com/docs/api/payments/downtime/#fetch-payment-downtime-details)
+
+-------------------------------------------------------------------------------------------------------
+
+### Fetch Payment Downtime
+
+```rb
+downtimeId = "down_F1cxDoHWD4fkQt"
+
+Razorpay::Payment.fetch_payment_downtime_by_id(downtimeId)
+```
+
+**Parameters:**
+
+| Name        | Type    | Description                          |
+|-------------|---------|--------------------------------------|
+| downtimeId* | string  | Id to fetch payment downtime         |
+
+**Response:**
+For payment downtime by id response please click [here](https://razorpay.com/docs/api/payments/downtime/#fetch-payment-downtime-details-by-id)
+-------------------------------------------------------------------------------------------------------
+
+### Payment capture settings API
+
+```rb
+para_attr = {
+  "amount":50000,
+  "currency": "INR",
+  "receipt": "rcptid_11",
+  "payment": {
+    "capture ": "automatic",
+    "capture_options ": {
+      "automatic_expiry_period ": 12,
+      "manual_expiry_period ": 7200,
+      "refund_speed": "optimum"
+    }  
+  }
+}
+Razorpay::Order.create(para_attr)
```

Comment:
what is this ? doesnt seem to be part of this pr

Reply:
We configure payment capture settings  with help of `order.create` api 

Doc : https://razorpay.com/docs/payments/payment-gateway/rainy-day/capture-settings/api/

Reply:
We configure payment capture settings  with help of `order.create` api 

Doc : https://razorpay.com/docs/payments/payment-gateway/rainy-day/capture-settings/api/